mycroft/skills/__main__.py

```diff
@@ -28,15 +28,14 @@
 from mycroft.api import is_paired, BackendDown, DeviceApi
 from mycroft.audio import wait_while_speaking
 from mycroft.enclosure.api import EnclosureAPI
-from mycroft.configuration import Configuration
+from mycroft.configuration import Configuration, set_default_lf_lang
```

Comment:
I have always been taught that this is an anti-pattern.  Why import it into `configuration.__init__` just to import it here?  This should be importing from mycroft.util.lang

Reply:
Assuming you mean directly from `mycroft.configuration.lang`?

This felt cleaner than doing two imports in each file:
```
from mycroft.configuration import Configuration
from mycroft.configuration.lang import set_default_lf_lang
```
But happy to change it.

Unless you meant - why not import directly from `lingua_franca.lang`? In which case the intention was to explicitly say that this is part of the mycroft-core API and it's acting on the methods available in mycroft-core.

Also for context - it was initially in `mycroft.util.lang` however even though it was to act on the `mycroft.util.format` and `parse` modules it felt like a configuration method, so I moved it over here. Again I'm not strongly attached to any particular implementation, this just felt better.